### PR TITLE
[aclorch] Enable IN_PORTS ACL qualifier on MIRROR tables

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

**What I did**
Opened the `IN_PORTS` qualifier for ACL tables.

**Why I did it**
To allow users to create ACL rules with `IN_PORTS` as a qualifier.

**How I verified it**
I added two new test cases to verify that the MIRROR table has all the correct qualifiers.
```
daall@valhalla:~/dev/sonic-swss/tests$ sudo pytest -s -v --dvsname=vs test_mirror_ipv6_separate.py 
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15+, pytest-3.3.0, py-1.8.0, pluggy-0.6.0 -- /usr/bin/python2
cachedir: .cache
rootdir: /home/daall/dev/sonic-swss/tests, inifile:
collected 6 items                                                                                                                                                                                                 

test_mirror_ipv6_separate.py::TestMirror::test_MirrorTableCreation PASSED                                                                                                                                   [ 16%]
test_mirror_ipv6_separate.py::TestMirror::test_MirrorV6TableCreation PASSED                                                                                                                                 [ 33%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorSeparated PASSED                                                                                                                                [ 50%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6Reorder1 PASSED                                                                                                                               [ 66%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6Reorder2 PASSED                                                                                                                               [ 83%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6WrongConfig PASSED                                                                                                                            [100%]

=========================================================================================== 6 passed in 119.31 seconds ============================================================================================
daall@valhalla:~/dev/sonic-swss/tests$ sudo pytest -s -v --dvsname=vs test_mirror_ipv6_combined.py 
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15+, pytest-3.3.0, py-1.8.0, pluggy-0.6.0 -- /usr/bin/python2
cachedir: .cache
rootdir: /home/daall/dev/sonic-swss/tests, inifile:
collected 5 items                                                                                                                                                                                                 

test_mirror_ipv6_combined.py::TestMirror::test_CombinedMirrorTableCreation PASSED                                                                                                                           [ 20%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Combined PASSED                                                                                                                               [ 40%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Reorder1 PASSED                                                                                                                               [ 60%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Reorder2 PASSED                                                                                                                               [ 80%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6WrongConfig PASSED                                                                                                                            [100%]

=========================================================================================== 5 passed in 119.91 seconds ============================================================================================
```

### CODE REVIEW

**Comment by daall** [comment]
retest this please

File: tests/test_mirror_ipv6_combined.py [hunk 1]
```diff
@@ -142,6 +142,78 @@ def remove_mirror_acl_rule(self, table, rule):
...
+        assert len(table_entries) == 1
+
+        # Get the data from the table
+        table_id = table_entries[0]
+        status, attributes = tbl.get(table_id)
+        assert status
+
+        # TODO: Refactor mirror table tests so that these attributes can be shared between tests for v4, v6, and
+        # dscp mirror tables.
+        expected_sai_attributes = [
```
Comment (yxieca) [inline tests/test_mirror_ipv6_combined.py]: I feel that 'expected' is a bit misleading to encourage people to think all attributes should be there? Or is it just my English? :-)

**Comment by daall** [comment]
retest this please

**Comment by daall** [comment]
retest this please

File: tests/test_mirror_ipv6_combined.py (hunk 1 above)
Comment (daall) [reply tests/test_mirror_ipv6_combined.py]: Thanks for catching this, I've added an extra check to make sure they're all there! :)

**Comment by daall** [comment]
retest this please

**Pull request merged**